### PR TITLE
docs: Update Azure migration guide styling

### PR DESCRIPTION
**Pull request by vdmgolub**

### CODE REVIEW

File: docs/guides/azure_devops_migration.md [hunk 1]
```diff
@@ -11,42 +11,42 @@ Follow this page to migrate from our old [pipeline bash scripts](https://github.
...
+- `update`: Create a single comment and update it on changes. This is the "quietest" option. For Azure DevOps Repos users, comments will simply be overwritten. GitHub users have additional UI that shows [what/when changed](https://docs.github.com/en/communities/moderating-comments-and-conversations/tracking-changes-in-a-comment) when the comment is updated. Pull request followers will only be notified on the comment create (not updates), and the comment will stay at the same location in the comment history.
 - `delete-and-new`: Delete previous cost estimate comments and create a new one. Pull request followers will be notified on each comment.
-- `hide-and-new`: Minimize previous cost estimate comments and create a new one. Pull request followers will be notified on each comment.
+- `hide-and-new`: Minimize previous cost estimate comments and create a new one. Pull request followers will be notified on each comment. This behavior is available only for GitHub.
```
Comment (hugorut) [inline docs/guides/azure_devops_migration.md:46]:
```suggestion

```

File: docs/guides/azure_devops_migration.md [hunk 2]
```diff
@@ -11,42 +11,42 @@ Follow this page to migrate from our old [pipeline bash scripts](https://github.
...
+🚀 The new [Infracost Azure DevOps Extension](https://marketplace.visualstudio.com/items?itemName=Infracost.infracost-tasks) provides a set of Azure Pipelines tasks offering a composable way of using Infracost in your pipeline. These JavaScript tasks simplify integrating Infracost into your Azure Pipelines. In addition, we've added Azure specific output formats, a cost summary table, and different behaviors, so you can control when comments are posted.
 
 ### Composable tasks
 
 The [Azure DevOps Extension](https://marketplace.visualstudio.com/items?itemName=Infracost.infracost-tasks) contains two main tasks:
 
-- InfracostSetup: install the Infracost CLI in your Azure Pipeline.
+- InfracostSetup: installs the Infracost CLI in your Azure Pipelines.
```
Comment (hugorut) [inline docs/guides/azure_devops_migration.md:20]:
```suggestion
- InfracostSetup: install the Infracost CLI in your Azure Pipeline.
```

Yes the product might be called `Azure Pipelines` but you install a task into a single **pipeline**

File: docs/guides/azure_devops_migration.md [hunk 3]
```diff
@@ -11,42 +11,42 @@ Follow this page to migrate from our old [pipeline bash scripts](https://github.
 
 ## What's new?
 
-🚀 The new [Infracost Azure DevOps Extension](https://marketplace.visualstudio.com/items?itemName=Infracost.infracost-tasks) provide a set of Azure Pipeline tasks offering a composable way of using Infracost in your pipeline. These JavaScript tasks simplify integrating Infracost into your Azure Pipeline. In addition, we've added Azure specific output formats, a cost summary table, and different behaviors, so you can control when comments are posted.
+🚀 The new [Infracost Azure DevOps Extension](https://marketplace.visualstudio.com/items?itemName=Infracost.infracost-tasks) provides a set of Azure Pipelines tasks offering a composable way of using Infracost in your pipeline. These JavaScript tasks simplify integrating Infracost into your Azure Pipelines. In addition, we've added Azure specific output formats, a cost summary table, and different behaviors, so you can control when comments are posted.
```
Comment (hugorut) [inline docs/guides/azure_devops_migration.md:14]:
```suggestion
🚀 The new [Infracost Azure DevOps Extension](https://marketplace.visualstudio.com/items?itemName=Infracost.infracost-tasks) provides a set of Azure Pipelines tasks offering a composable way of using Infracost in your pipeline. These JavaScript tasks simplify integrating Infracost into your Azure Pipeline. In addition, we've added Azure specific output formats, a cost summary table, and different behaviors, so you can control when comments are posted.
```
Product name: `Azure Pipelines` but a single entity is  `Pipeline`

File: docs/guides/azure_devops_migration.md [hunk 4]
```diff
@@ -11,42 +11,42 @@ Follow this page to migrate from our old [pipeline bash scripts](https://github.
...
+The InfracostComment task includes a `behavior` and a `targetType` attributes.
 
 Behavior describes how and when comments should be posted; we support four options:
-- `update`: Create a single comment and update it on changes. This is the "quietest" option. For Azure DevOps Repos users, comments will simply be overwritten. GitHub users have additional UI that show [what/when changed](https://docs.github.com/en/communities/moderating-comments-and-conversations/tracking-changes-in-a-comment) when the comment is updated. Pull request followers will only be notified on the comment create (not updates), and the comment will stay at the same location in the comment history.
+- `update`: Create a single comment and update it on changes. This is the "quietest" option. For Azure DevOps Repos users, comments will simply be overwritten. GitHub users have additional UI that shows [what/when changed](https://docs.github.com/en/communities/moderating-comments-and-conversations/tracking-changes-in-a-comment) when the comment is updated. Pull request followers will only be notified on the comment create (not updates), and the comment will stay at the same location in the comment history.
```
Comment (vdmgolub) [inline docs/guides/azure_devops_migration.md:44]: It seems like "the quietest option" is not true for Azure Repos. It sends an email notification with "base Build Service (infracost) has edited a comment" :)

**Review by hugorut — CHANGES_REQUESTED** [review]
I'm not sure I entirely agree with the renaming of all the instances of `Azure Pipeline` to `Pipelines`. The product name is called `Pipelines` as it refers to a set of CI pipelines that the user runs. A single instance of this would be a Pipeline. It's much easier to refer to a pipeline in the singular when referring to things like `jobs` and `stages` - microsoft does this in their docs: https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema So I think we should stick with that.

File: docs/guides/azure_devops_migration.md [hunk 5]
```diff
@@ -58,10 +58,10 @@ The `target-type` describes where the comment should be posted against, which ca
    - [Terragrunt](https://github.com/infracost/infracost-azure-devops/examples/terragrunt): a Terragrunt project
    - [Terraform Cloud/Enterprise](https://github.com/infracost/infracost-azure-devops/examples/terraform-cloud-enterprise): a Terraform project using Terraform Cloud/Enterprise
    - [Multi-project using config file](https://github.com/infracost/infracost-azure-devops/examples/multi-project/README.md#using-an-infracost-config-file): multiple Terraform projects using the Infracost [config file](https://www.infracost.io/docs/multi_project/config_file)
-   - [Multi-project using ajob matrix](https://github.com/infracost/infracost-azure-devops/examples/multi-project/README.md#using-azure-devops-pipeline-matrix-strategy): multiple Terraform projects using Azure Pipeline job matrix strategy
+   - [Multi-project using ajob matrix](https://github.com/infracost/infracost-azure-devops/examples/multi-project/README.md#using-azure-devops-pipeline-matrix-strategy): multiple Terraform projects using Azure Pipelines job matrix strategy
```
Comment (hugorut) [inline docs/guides/azure_devops_migration.md:61]:
```suggestion
   - [Multi-project using ajob matrix](https://github.com/infracost/infracost-azure-devops/examples/multi-project/README.md#using-azure-devops-pipeline-matrix-strategy): multiple Terraform projects using Azure Pipeline job matrix strategy
```

File: docs/guides/azure_devops_migration.md (hunk 3 above)
Comment (vdmgolub) [reply docs/guides/azure_devops_migration.md:14]: > This article is a detailed reference guide to Azure Pipelines YAML pipelines

It's the first line of their documentation, looks mouthful to me :) Do we want to capitalize the "Pipeline" then? It's not the name anymore.

File: docs/guides/azure_devops_migration.md (hunk 1 above)
Comment (vdmgolub) [reply docs/guides/azure_devops_migration.md:46]: Do we want to remove it completely, even for GitHub users?

Comment (hugorut) [reply docs/guides/azure_devops_migration.md:46]: ah sorry I misunderstood - no let's keep it.

File: docs/guides/azure_devops_migration.md (hunk 3 above)
Comment (hugorut) [reply docs/guides/azure_devops_migration.md:14]: yeah might be best just to lowercase it

**Review by hugorut — APPROVED** [review]

**Pull request merged**